src/openrct2/rct1/S4Importer.cpp

```diff
@@ -1489,7 +1489,8 @@ class S4Importer final : public IParkImporter
         dst->Id = src->id;
         dst->CashInPocket = src->cash_in_pocket;
         dst->CashSpent = src->cash_spent;
-        dst->TimeInPark = src->time_in_park;
+        // This doubles as staff hire date
+        dst->ParkEntryTime = src->park_entry_time;
```

Comment:
I'd find it more consistent of this also used a `union`.

@duncanspumpkin would that be OK with you?

Reply:
I don't like to modify the rct1/2 structs much except to update the naming. If you feel it would be useful then go ahead. I think eventually I'm going to want to split the import code into a staff and a guest import so it would help with that.

Reply:
There's a few more union fields in Peep that are not union in the rct1/2 structs (eg voucher_ride_id, paid_on_souvenirs, paid_on_food)...should I make them all union? I can also split the import/export if we want to do so now, or are we waiting for the new save format.